organizations/user.go

```diff
@@ -171,3 +171,37 @@ func UpdateProfilePicture(w http.ResponseWriter, r *http.Request) {
 	utils.GetSuccess("Profile picture updated", result, w)
 
 }
+
+func GetProfile(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+	org_collection, member_collection := "organizations", "members"
+
+	Id := mux.Vars(r)["id"]
+	MemId := mux.Vars(r)["mem_id"]
+
+	OrgId, err := primitive.ObjectIDFromHex(Id)
+	if err != nil {
+		utils.GetError(errors.New("invalid id"), http.StatusBadRequest, w)
+		return
+	}
+
+	orgDoc, _ := utils.GetMongoDbDoc(org_collection, bson.M{"_id": OrgId})
+	if orgDoc == nil {
+		utils.GetError(errors.New("Organnization id doesn't exist!"), http.StatusBadRequest, w)
+		return
+	}
+
+	MemberId, err := primitive.ObjectIDFromHex(MemId)
+	if err != nil {
+		utils.GetError(errors.New("invalid id"), http.StatusBadRequest, w)
+		return
+	}
+
+	MemberProfile, err := utils.GetMongoDbDoc(member_collection, bson.M{"_id": MemberId})
```

Comment:
You are retrieving an entire member collection and not the profile data of a member. Please check "edit profile" feature in slack and see the fields you need to extract from the member collection. When you retrieve a doc, it has all the fields in it in a map form e.g MemberProfile["name"] will have the name of the member, you can extract all the data that make up a "profile", then put then in a profile struct for example and send that back as a response. The struct can also be used to update a member profile too.